test/ansible/slc_setup_test.yml

```diff
@@ -0,0 +1,24 @@
+- name: Prepare environment
+  hosts: localhost
+  gather_facts: false
+  vars:
+      ansible_python_interpreter: python3
+
+  tasks:
+
+    - name: add docker container to inventory
```

Comment:
Do we need this , thought we wanted to remove it?

Reply:
This is just a convenience to add the docker container to the repository. The docker container itself is started via the Docker SDK. 
I could manipulate the inventory by hand, but I found this method more safe and easy to maintain.